src/pages/StatusPage/index.tsx

```diff
@@ -3,69 +3,86 @@ import { PageStatus, StatusRow } from '../../components';
 import { begin } from '../../services';
 import { Api, ApiWithResponse, Response } from '../../typings/Api';
 import { Container } from './StatusPage.elements';
-
+ 
 interface StatusPageProps {
-  /** An array of API objects */
-  apis: Api[];
-  /** The interval at which to call the APIs in milliseconds */
-  interval?: number;
+ /** An array of API objects */
+ apis: Api[];
+ /** The interval at which to call the APIs in milliseconds */
+ interval?: number;
 }
-
+ 
 interface ApiWithError extends ApiWithResponse {
-  hasError: boolean;
+ hasError: boolean;
 }
-
+ 
 interface StatusPageState {
-  apis: ApiWithError[];
-  hasError: boolean;
+ apis: ApiWithError[];
+ hasError: boolean;
 }
-
+ 
 class StatusPage extends Component<StatusPageProps> {
-  public state: StatusPageState = {
-    apis: [],
-    hasError: false,
-  };
-
-  public handleError = (api: Api, response: Response) => {
-    const apis = [...this.state.apis];
-    const index = apis.findIndex(stateApi => stateApi.api === api);
+ public state: StatusPageState = {
+   apis: [],
+   hasError: false,
+ };
+ 
+ 
+ public handleResponse = (api: Api, response: Response) => {
```

Comment:
Still need a `handleError` - see my comment below on the line that calls `begin`

Reply:
See above